### PR TITLE
modify event.get_station() to return the first and only station of no…

### PR DESCRIPTION
**Pull request by fschlueter**
… station id is given

The idea is to avoid code like:

```
for station in evt.get_stations():
   ....
```
or worse 
```
station = evt.get_station(evt.get_station_ids()[0])
```
if it is clear that the event has only one station (which is typically the case ?!)

What do you think about that? Unnecessary?

### CODE REVIEW

File: NuRadioReco/framework/event.py [hunk 1]
```diff
@@ -128,7 +128,30 @@ def set_id(self, evt_id):
     def get_run_number(self):
         return self.__run_number
 
-    def get_station(self, station_id):
+    def get_station(self, station_id=None):
+        """
+        Returns the station for a given station id.
+
+        Paramters
```
Comment (sjoerd-bouma) [inline NuRadioReco/framework/event.py:135]: Parameters

File: NuRadioReco/framework/event.py [hunk 2]
```diff
@@ -128,7 +128,30 @@ def set_id(self, evt_id):
...
+        ---------
+
+        station_id: int
+            Id of the station you want to get. If None and event has only one station
+            return it, otherwise raise error. (Default: None)
+
+        Return
```
Comment (sjoerd-bouma) [inline NuRadioReco/framework/event.py:142]: Returns

**Review by sjoerd-bouma — CHANGES_REQUESTED** [review]
I'm all for quality of life improvements, and I guess this shouldn´t break any backwards compatibility. Please do correct the docstring headings, as they will cause errors in compiling the documentation otherwise (which I guess is hard to see at the moment because the tests fail in any case...)

File: NuRadioReco/framework/event.py (hunk 2 above)
Comment (fschlueter) [reply NuRadioReco/framework/event.py:142]: Even if we return a single station :o ;)

**Review by sjoerd-bouma — APPROVED** [review]
I've double checked that the documentation compiles locally, seeing as the tests don't work currently

**Pull request merged**